package.json

```diff
@@ -124,36 +125,39 @@
     "redux-raven-middleware": "^1.2.0",
     "redux-thunk": "^2.3.0",
     "sass": "^1.62.1",
-    "sass-loader": "^7.0.0",
+    "sass-loader": "^13.3.2",
     "scrolltop": "0.0.1",
     "simple-git": "^3.3.0",
     "snyk": "^1.695.0",
-    "style-loader": "^2.0.0",
-    "terser-webpack-plugin": "^1.4.5",
+    "style-loader": "^3.3.3",
+    "terser-webpack-plugin": "^5.2.5",
```

Comment:
Seems there is a problem with the terser plugin when building Docker image:
`ValidationError: Invalid options object. Terser Plugin has been initialized using an options object that does not match the API schema.
#0 17.61  - options has an unknown property 'sourceMap'. These properties are valid:
#0 17.61    object { test?, include?, exclude?, terserOptions?, extractComments?, parallel?, minify? }...`

Reply:
Fixed